src/ui/styles/colors.styl

```diff
@@ -1,4 +1,5 @@
 $black = #000;
+$deepGray = #292f3c;
```

Comment:
I wouldn't bother adding more color variables here. I guess these are from mockups mainly. And AFAIK we're going to get rid of stylus anyway.